userguide/kaui/Kaui_Guide_Draft.adoc

```diff
@@ -889,92 +889,203 @@ To delete a payment method:
+==== Invoice Details Page Layout
 
-Kill Bill _does not_ automatically generate an invoice payment if:
+The Invoice Details page displays information about the invoice, such as the date it was generated, amount, and so forth.
 
-* No default payment method exists for the account
-* The default payment method is invalid (e.g., expired credit card)
-* The `AUTO_PAY_OFF` or `MANUAL_PAY` https://docs.killbill.io/latest/userguide_subscription.html#components-tag[system tags] are defined for the acocunt
+image::Invoice_Detail.png[width=650,align="center"]
 
-MORE ON THIS? --> An example of this would be a check payment from the customer.
+On the Invoice Detail page, you can:
 
-==== Make a Payment on an Invoice
+* Make a payment against an invoice
+* Write off or void an invoice
+* See the payment details (if the invoice was paid)
 
-To make a manual payment against an invoice:
+[NOTE]
+*Note:* You can perform two other invoice-related tasks on the Accounts page: <<Generate a Dry-Run Invoice>> and <<Pay All Unpaid Invoices>>.
 
-1. Open the account on the Account page.
-2. Click on Invoices. Kaui displays the Invoices page:
+==== Open Invoice Details
 
-image::Invoices.png[width=650,align="center"]
+To get to the Invoice Details page:
 
-[start=3]
-. Click on the number of the invoice that needs to be paid off. This opens the invoice detail.
+. Open an account.
+. Click on the *Invoices* sub-menu.
+. Select an invoice number.
+
+[NOTE]
+*Note:* To see any associated payment details, you may need to scroll down the page.
+
+==== Make a Payment on an Invoice
+
+To make a manual payment against an invoice:
+
+1. <<Open Invoice Details, Open the Invoices Details page>> for the invoice.
 
 image::Invoice_Detail.png[width=650,align="center"]
 
-[start=4]
+[start=2]
 . Hover over the credit card icon and select *Make Payment*.
 
 image::Invoice_MakePayment.png[width=350,align="center"]
 
 [start=5]
 . Kaui displays the Process Payment screen:
 
-image::Invoice_ProcessPayment.png[width=650,align="center"]
+image::Invoice_ProcessPayment.png[width=450,align="center"]
 
 [start=6]
 . Fill in the fields. For field descriptions, see the table in the next section.
-. Click *Save*. The invoice detail shows the balance due on the invoice. Below the invoice, Kaui displays the payment details (e.g., status of the payment).
+. Click *Save*. The invoice detail shows the balance due on the invoice. Below the invoice, Kaui displays the payment details, including the status of the payment.
 
 ==== Process Payment Field Descriptions
 
 [cols="1,3"]
 |===
 ^|Field ^|Description
 
-|External
-|Select this checkbox if the customer is making a payment outside of the Kill Bill system (such as with a check).
+|External?
+|Select this checkbox if the customer is making a payment outside of the Kill Bill system (such as with a check). Note that if this checkbox is selected, it overrides anything selected in the *Payment Method* field.

```

Comment:
Typo: Question mark after `External`

Reply:
That's what the field label actually is, but I can remove it if you think it's confusing/distracting.

Reply:
oh okay, then I guess we should leave it as it is, I thought it was a typo and did not want it to do unnoticed :)